Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -786,15 +792,21 @@ void ABenchmarkGymGameModeBase::ReportMigration_Implementation(const FString& Wo
 	}
 }
 #if	STATS
-void ABenchmarkGymGameModeBase::SetStatTimer(const FString& TimeString)
+void ABenchmarkGymGameModeBase::InitStatTimer(const FString& CPUProfileString)
 {
-	FString StartDelayString, DurationString;
-	if (TimeString.Split(TEXT(","), &StartDelayString, &DurationString))
+	FString CPUProfileIntervalString, CPUProfileDurationString;
+	if (CPUProfileString.Split(TEXT("&"), &CPUProfileIntervalString, &CPUProfileDurationString))
```

Comment:
Could we document this format please? Maybe it's worth having some info in a comment block somewhere describing the options we have available for profiling